pydis_core/utils/logging.py

```diff
@@ -9,6 +9,12 @@
     LoggerClass = logging.getLoggerClass()
 
 TRACE_LEVEL = 5
+log_format = logging.Formatter("%(asctime)s | %(levelname)s | %(name)s | %(message)s")
+"""
+A :obj:`logging.Formatter` that can be used to standardise logging across all projects using pydis_core.
+
+:meta hide-value:
```

Comment:
Without this line the docs look like this, which doesn't look good in my opinion
![image](https://github.com/python-discord/bot-core/assets/9753350/71b3aaeb-8ccc-41b3-81c2-32b7922dc07e)